.packer/ami-version.txt

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
We can probably pass this to the name of the AMI via the options block in the template :thinking: